tox.ini

```diff
@@ -20,6 +20,7 @@ deps =
     pip19.3: -e git+https://github.com/pypa/pip.git@master#egg=pip
     mock
     pytest!=5.1.2
+    pytest-rerunfailures
     coverage: pytest-cov
 setenv =
```

Comment:
Coverage is decreased because `passenv = CI` is missing in tox.ini in [testenv] section.

Reply:
Thanks, I'm working on it today